boards/omnigul/omnigul.config

```diff
@@ -0,0 +1,77 @@
+# Configuration for a Acer Chromebook Plus 515 (CB515-2H/OMNIGUL)
+# Intel iGPU "UHD Graphics", 12th Gen Intel Core i3-1215U, 8GB RAM
+# 128GB UFS Storage (iNAND brand), Intel AX211 Wi-FI 6E,
+# 32MB Winbond Chip W25Q256JVEM - WSON8 8x6 probe needed.
+
+export CONFIG_COREBOOT=y
+export CONFIG_COREBOOT_VERSION=mrchromebox
+export CONFIG_LINUX_VERSION=6.6.30
+
+CONFIG_COREBOOT_CONFIG=config/coreboot-omnigul.config
+CONFIG_LINUX_CONFIG=config/linux-omnigul.config
+
+#Enable DEBUG output
+#export CONFIG_DEBUG_OUTPUT=y
+#export CONFIG_ENABLE_FUNCTION_TRACING_OUTPUT=y
+#Enable TPM2 pcap output under /tmp
```

Comment:
@mdrobnak until this port is successful, I think you might want to add a commit turning this on and a final one turning this off